.eslintignore

```diff
@@ -10,3 +10,6 @@ source/common/assets/tpl
 # Ignore jQuery UI and the copied indentlist-addon
 source/common/assets/js/jquery-ui.min.js
 source/renderer/assets/codemirror/indentlist.js
+
+# Ignore build output
+dist
```

Comment:
I would argue we can directly stuff the TypeScript output into the correct directories?

Reply:
It is possible to emit the compiled javascript files into source. But then you run into problems with git and the other post-processing steps.  I think this is actually cleaner this way: have all source files of whatever type (js, ts, css, img, ...) in the "source" folder and upon compilation these source files are processed (by tsc, webpack, gulp, whatever) to yield the correct "compiled" version in the "dist" folder.
This way you get a clear and clean distinction between original source files and compiled files.

Reply:
I see what you mean, but first, the `source`-directory is the one `electron-builder` expects the files to be in and we can add corresponding gitignore-rules, which work for the vue sources easily.

So as it is now it's easier to include into the recommended electron-workflow (plus we still would have a separation of source and compiled files). My aim here currently is to have the source-directory only contain compiled files and assets in the future, and have all the "real" source files in resources, maybe `lib` (which would be the TypeScript equivalent, if I'm not mistaken?). We can discuss the file structure, though!

Reply:
Electron-forge should make this pretty convenient: all the source files are put into /source (or the somewhat more common /src by default) and forge then takes care of the proper post-processing (and puts everything in the /.webpack folder). I'll pause this PR for now until the migration to forge is complete.